### PR TITLE
field injection changed to constructor injection for autowiring

### PR DESCRIPTION
**Pull request by ashimsharma337**
I changed the field injection to constructor injection in controller and service class of reservation, and issues with put request fixed

### CODE REVIEW

**Review by edymol — COMMENTED** [review]
Please, use original password for database before committing.. MyPassword123

File: restaurant/src/main/resources/application.properties
Comment (edymol) [inline restaurant/src/main/resources/application.properties:1]: Change this file to original MyPassword123

File: restaurant/target/classes/application.properties
Comment (edymol) [inline restaurant/target/classes/application.properties:1]: Change this file to original MyPassword123

**Review by edymol — CHANGES_REQUESTED** [review]
Change the password please to MyPassword123, I don't want to rewrite this change every time someone works on it because a new commit will be created

File: restaurant/target/classes/application.properties
Comment (ashimsharma337) [reply restaurant/target/classes/application.properties:1]: Sorry, I changed it now.

**Review by edymol — APPROVED** [review]

**Review by edymol — APPROVED** [review]

**Review by edymol — APPROVED** [review]

**Review by edymol — APPROVED** [review]

**Pull request closed without merging**